DeviceComunication/DeviceComunication_CZ.md

```diff
@@ -1048,3 +1092,9 @@ Zařízení vypočítá počet měření, které je potřeba udělat před odesl
 
 Zařízení pak bude čekat mezi měřeními 4 minuty a po každých dvou měření odešle zprávu. Measure
 zprávy se tedy budou odesílat každých 8 minut.
+
+## Zjednodušená implementace potvrzování
+Protože zařízení ve výchozím stavu vyžaduje potvrzení některých zpráv 
+(určuje [5.byte v hlavičce](#5byte---nb-iot)), je nutné implementovat alespoň 
+základní komunikaci ze serveru do zařízení. Na tyto zprávy je možné zjednodušeně 
+odpovídat staticky payloadem `0000000001FF00` který zajistí potvrzení přijaté zprávy.
```

Comment:
Nikde jsem nenasel tu zminku o tom ze Lora zpravy se mohou ztratit.  Prehlednul jsem to?

Reply:
Je tam upraveny text v závorce - "Pro LoRa síť nemá význam odesílat na zařízení potvrzovací zprávu, ikdyž si ji zařízení vyžádá, jelikož o potvrzení se stará LoRa síť automaticky (u zpráv vyžadujících potvrzení)."